### PR TITLE
Update to SRB2 2.2.13

### PR DESCRIPTION
**Pull request by LeonardoTheMutant**
## Update to SRB2 2.2.13

- [x] Tested with ***SRB2 2.2.13 "Speedrun edition"*** (server) + ***SRB2 2.2.13 (Vanilla)*** (client)
- [ ] Untested with Website's API (but the server loads records from Website's database to client's SCORES TAB if `show_highscores.lua` is added)
- [X] Server communicates with MySQL database on `localhost`
- [X] Tested on **Linux Mint 21.2** (SRB2 & MYSQL server via VM) & **Windows 7** (SRB2 client)

As far as I believe this update will work without issues on a real *srb2circuit.eu* server

### CODE REVIEW

**Comment by JJK96** [comment]
@LeonardoTheMutant what do you mean with the untested part, because when I read it, it seems that you tested that the functionality works?

File: src/speedrun.c [hunk 1]
```diff
@@ -214,17 +221,18 @@ void speedrun_map_completed()
...
+    for (unsigned short playernum = 0; playernum < MAXPLAYERS; playernum++) {
         // if the player is not in game, is a spectator or is dead: skip it
-    	if (!playeringame[playernum] 
-        	  || players[playernum].spectator 
-        	  || players[playernum].pflags & PF_GAMETYPEOVER 
-        	  || players[playernum].laps < (unsigned)cv_numlaps.value)
+    	if (!playeringame[playernum]
+		|| players[playernum].spectator
+		|| players[playernum].pflags & PF_GAMETYPEOVER
+		|| players[playernum].laps < (unsigned)cv_numlaps.value) //ruins calculation for all non-circuit maps
```
Comment (JJK96) [inline src/speedrun.c]: What do you mean by this comment?

File: src/speedrun.h.gch
Comment (JJK96) [inline src/speedrun.h.gch:1]: I don't think you want/need to push this binary file

**Comment by LeonardoTheMutant** [comment]
@JJK96 I tried to build only SRB2 + MySQL part on my local machine and never expected that SRB2 would load the data from the website

**Comment by JJK96** [comment]
Yeah, the website API is publicly accessible, so it can indeed load the data from there

**Pull request closed without merging**